ie-comunicaciones.csl

```diff
@@ -0,0 +1,205 @@
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <group>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <bibliography hanging-indent="true" et-al-min="6" et-al-use-first="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+      <key variable="title"/>
+    </sort>
+    <layout>
+      <group delimiter="," prefix="[" suffix="] ">
+        <group>
+          <text macro="author-short" suffix=" "/>
+          <text macro="year-date"/>
+        </group>
+      </group>
+      <group>
+        <text macro="author" suffix="."/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+            <group prefix=" " delimiter=" " suffix=".">
+              <text macro="title" quotes="false"/>
+              <text macro="edition"/>
+              <text macro="editor"/>
+            </group>
+            <text prefix=" " suffix="." macro="publisher"/>
+          </if>
+          <else-if type="chapter paper-conference" match="any">
+            <text macro="title" prefix=" " suffix="."/>
+            <group prefix=" " delimiter=" ">
+              <text term="in" text-case="capitalize-first"/>
+              <text macro="editor"/>
+              <group suffix="," delimiter=".">
+                <text variable="container-title" font-style="italic"/>
+                <text variable="collection-title"/>
+              </group>
+              <text macro="pages" suffix="."/>
+              <text macro="publisher"/>
+            </group>
+          </else-if>
+          <else-if type="thesis">
+            <group prefix=" " suffix="." delimiter=". ">
+              <text macro="title"/>
+              <text variable="genre"/>
+              <text macro="publisher"/>
+            </group>
+          </else-if>
+          <else>
+            <group suffix=".">
+              <text macro="title" quotes="false" prefix=" "/>
+              <text macro="editor" prefix=" "/>
+            </group>
+            <group prefix=" ">
+              <text variable="container-title" font-style="normal" suffix=","/>
+              <group>
+                <group prefix=" " suffix=",">
+                  <group>
+                    <text term="volume" form="symbol" text-case="capitalize-first" suffix=" "/>
+                    <text variable="volume"/>
+                  </group>
+                  <group>
+                    <text term="issue" form="symbol" text-case="capitalize-first" prefix=" "/>
+                    <text variable="issue" prefix=" "/>
+                  </group>
+                </group>
+                <group prefix=" " suffix=".">
+                  <label plural="always" variable="locator" form="short"/>
```

Comment:
Actually, I'm just guessing with the always plural. Because frankly their directions for authors are quite incomplete and in the example they use plural. 

However I'm changing it to contextual.